### PR TITLE
Hide private groups/projects

### PR DESCRIPTION
**Pull request by Gurrman375**
Now hides private and group presences:
Shown:
image
image

Hidden:
image

### CODE REVIEW

**Comment by ririxidev** [comment]
That was fast response to my comment! Thanks!
But please fix ESLint issues.

**Review by Slowlife01 — CHANGES_REQUESTED** [review]
ESLint issues

File: websites/G/GitLab/presence.ts [hunk 1]
```diff
@@ -2,84 +2,55 @@ const presence = new Presence({
...
-    else if (document.location.pathname == "/search") {
+      if (document.location.pathname === "/dashboard/projects/starred")presenceData.details = "Viewing Starred";
+      else presenceData.details = "Viewing Dashboard";
+    } else if (document.location.pathname === "/projects/new/")presenceData.details = "Creating a project";
+     else if (document.location.pathname === "/groups/new")presenceData.details = "Creating a group";
+     else if (document.location.pathname.includes("/explore")) {
+      if (document.location.pathname === "/explore/snippets") presenceData.details = "Exploring Snippets";
+      else presenceData.details = "Exploring projects";
+    } 
+    else if (document.location.pathname === "/dashboard/groups") presenceData.details = "Viewing Groups";
```
Comment (ririxidev) [inline websites/G/GitLab/presence.ts:25]:
```suggestion
    } else if (document.location.pathname === "/dashboard/groups") presenceData.details = "Viewing Groups";
```

File: websites/G/GitLab/presence.ts [hunk 2]
```diff
@@ -2,84 +2,55 @@ const presence = new Presence({
...
+
+    else if (document.location.pathname === "/dashboard/merge_requests") presenceData.details = "Viewing Merge Requests";
+
+    else if (document.location.pathname === "/dashboard/todos") presenceData.details = "Viewing TODOS";
+
+    else if (document.location.pathname === "/help") presenceData.details = "Viewing Help";
+
+    else if (document.location.pathname === "/users/sign_in") presenceData.details = "Signning in or Registering";
+
+
```
Comment (ririxidev) [inline websites/G/GitLab/presence.ts:49]:
```suggestion
```

File: websites/G/GitLab/presence.ts [hunk 3]
```diff
@@ -91,96 +62,68 @@ presence.on("UpdateData", async () => {
...
-        presenceData.state = owner.innerText;
-        presenceData.startTimestamp = browsingStamp;
-      } else {
-        presenceData.startTimestamp = browsingStamp;
-        presenceData.details = "Viewing Unknown";
+      if((presenceprivate as HTMLTextAreaElement).title.startsWith("Private -") && lock) {
+        presenceData.details = "Viewing a Private Presence";
+        presenceData.state = "or Private Group";
+      }
+      else{
```
Comment (ririxidev) [inline websites/G/GitLab/presence.ts:69]:
```suggestion
      } else {
```

**Review by rxri — CHANGES_REQUESTED** [review]

**Review by rxri — APPROVED** [review]

**Review by Slowlife01 — APPROVED** [review]

**Pull request merged**